summoner-cli/src/Summoner/Source.hs

```diff
@@ -33,18 +34,22 @@ showSource = \case
     File _ -> "File"
     Link _ -> "Link"
```

Comment:
Not relevant to this PR, but I think `Link` field deserves better documentation, as even I required some time to remember where and why this could be used :smile:

Reply:
Yes! I actually completely forgot that we have `Link` constructor. I thought that we have `Content` constructor where you can specify the full content of the file. Since TOML support multi-line strings in nice syntax, we can add `Content` constructor later and automatically support templating feature.

Reply:
Wow! This is awesome!